Sources/Sentry/SentryOptions.m

```diff
@@ -279,6 +279,10 @@ - (BOOL)validateOptions:(NSDictionary<NSString *, id> *)options
 
     [self setBool:options[@"sendClientReports"]
             block:^(BOOL value) { self->_sendClientReports = value; }];
+    
+    if ([options[@"sdk"] isKindOfClass:[NSDictionary class]]) {
```

Comment:
`h`: I think this should be `sdkInfo`:
```suggestion
    if ([options[@"sdkInfo"] isKindOfClass:[NSDictionary class]]) {
```

Reply:
I agree and would have called it that (and passed the directory content, not the whole options), but the SdkInfo already expect this dictionary structure, including the "sdk": https://github.com/getsentry/sentry-cocoa/blob/d68e2b6e0e62b0e26fede487eb5c2642825fa3fe/Sources/Sentry/SentrySdkInfo.m#L34

So would you prefer the options would look like this?
```
options: {
  sdkInfo: {
    sdk: {
      name: ...
      version
    }  
  }
}
```

Reply:
Sorry I missed that. Then maybe worth a comment

Reply:
added the comment